CMakeLists.txt

```diff
@@ -80,7 +80,7 @@ add_executable(osrm-extract src/tools/extract.cpp)
 add_executable(osrm-contract src/tools/contract.cpp)
 add_executable(osrm-routed src/tools/routed.cpp $<TARGET_OBJECTS:SERVER> $<TARGET_OBJECTS:UTIL>)
 add_executable(osrm-datastore src/tools/store.cpp $<TARGET_OBJECTS:UTIL>)
-add_library(osrm src/osrm/osrm.cpp $<TARGET_OBJECTS:ENGINE> $<TARGET_OBJECTS:UTIL>)
+add_library(osrm src/osrm/osrm.cpp $<TARGET_OBJECTS:ENGINE> $<TARGET_OBJECTS:UTIL> $<TARGET_OBJECTS:STORAGE>)
```

Comment:
ENGINE now depends on STORAGE, but we do not specify that dependency.

For example: https://gist.github.com/daniel-j-h/c0310687bf9664a7bbf1

Required me to add STORAGE to the engine tests, too, because ENGINE now depends on STORATGE in https://github.com/Project-OSRM/osrm-backend/pull/2088/commits/9f9ec2a45c830f01a9899b7d7d0d126b8cdf00c1.